src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,4 +12,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
 
+    public static final Prefix PREFIX_CLASS = new Prefix("/c");
```

Comment:
Should we standardise the prefixes to be "/" before the letter or after?

Reply:
In our user guide we put "/" before so I guess we can keep it that way